.gitignore

```diff
@@ -21,3 +21,4 @@ debian/delphix-platform.debhelper.log
 debian/delphix-platform.substvars
 debian/delphix-platform
 debian/files
```

Comment:
we should have `debian/tmp` listed here too.

this is unrelated to this change, but the 3 `delphix-platform` files listed here should actually be the 3 files specific to each platform's package.. e.g.
```
debian/delphix-platform-kvm.debhelper.log
debian/delphix-platform-kvm.substvars
debian/delphix-platform-kvm
```
we must have left off this change when we moved to the platform specific packages.